### ISSUE TITLE
Multiple Time Signatures across multiple tracks

### ISSUE THREAD

**Issue by dgriessler** [CoderLine/alphaTab#287]
# Your environment

* Version used: (master branch)
master
* Platform used: JavaScript or C#
JavaScript
* Rendering engine used: SVG (default), HTML5 or GDI
SVG
* Browser Name and Version: 
Firefox 69.0.2
* Operating System and version (desktop or mobile):
Windows x64 desktop
* Link to your project:
Github: https://github.com/dgriessler/TestHost/tree/gh-pages (for source code)
Website: https://dgriessler.github.io/TestHost (for rendering)

# Expected Results

I have multiple tracks and one set of tracks has a time signature 4 4 and the other set of tracks has a time signature 12 8.  The tracks with a time signature 4 4 should have that displayed on their line and those with 12 8 should have that displayed on their line.

# Observed Results

The 12 8 time signature is displayed on all tracks.

# Steps to Reproduce (for bugs)

1. See the linked source code.  The index.html contains the alphaTex currently being rendered.
In a nutshell:
```
\track "Soprano"
\staff {score} \tuning piano \instrument acousticgrandpiano \ks G
\ts 4 4 :2 g3{-} r |
\track "Piano Accompaniment"
\staff {score} \tuning piano \instrument acousticgrandpiano \ks G \clef F4
\ts 12 8 :1 g2{d} |
```
will show only the 12 8 time signature.  The alphaTex that is currently being rendered is longer.  

# Possible Solution 

I'm guessing that it assumes there is only one time signature across tracks.  You'd have to expand it to accept multiple and draw multiple.

**Comment by Danielku15** [comment]
I think this is not how music sheets work. Time signatures are just like the tempo an element that must be the same for all instruments placed within a song. It would break almost any logic when you display 2 instruments of the same song side-by-side.

**Comment by dgriessler** [comment]
It's called Polytempo https://en.wikipedia.org/wiki/Polytempo
Although rare for it to occur in one track, some composers do include it.  In my case, the piano accompaniment is in 12 8 while the choral group is singing in 4 4.  I can submit a feature request if that should be my next step since it doesn't sound like its currently supported.

**Comment by Danielku15** [comment]
I lately found out how this is handled in MuseScore and we might adopt the same logic: 

* There is a general time signature definition for the masterbars which apply normally to all bars
* Individual tracks can specify overrides of time signatures. 
* The override time signatures are purely visual for rendering. Actual playback is not affected by it in any way. The cursor display is at best-effort assuming the timely placement will fit across signatures. Meaning: it is assumed that a 2/4 is overridden by a 4/4 but not a 3/4 which would mean a different total bar duration.

**Comment by dgriessler** [comment]
Sweet, sounds reasonable to me.
